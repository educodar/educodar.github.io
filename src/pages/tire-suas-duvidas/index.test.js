import React from "react"
import renderer from "react-test-renderer"

import FAQPage from "."

describe("FAQ page", () => {
  /**
   * @todo fix failing test FAQPage (...): Nothing was returned from render
   */
  xit("renders correctly", () => {
    const tree = renderer.create(<FAQPage />)
    expect(tree).toMatchSnapshot()
  })
})
