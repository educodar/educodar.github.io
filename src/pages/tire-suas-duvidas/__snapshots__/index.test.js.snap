// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQ page renders correctly 1`] = `
<article
  className="MuiBox-root MuiBox-root-2"
>
  <header
    className="MuiBox-root MuiBox-root-3"
  >
    <h1
      className="MuiTypography-root MuiTypography-h1 MuiTypography-gutterBottom"
    >
      Tire suas dúvidas
    </h1>
  </header>
  <dl
    className="MuiBox-root MuiBox-root-34"
  >
    <div
      className="MuiBox-root MuiBox-root-35"
      itemProp="mainEntity"
      itemScope={true}
      itemType="https://schema.org/Question"
    >
      <dt
        className="MuiBox-root MuiBox-root-36"
      >
        O que é o Educodar
      </dt>
      <dd
        className="MuiBox-root MuiBox-root-37"
        itemProp="acceptedAnswer"
        itemScope={true}
        itemType="https://schema.org/Answer"
      >
        <div
          className="MuiBox-root MuiBox-root-38"
          itemProp="text"
        >
          É um projeto voluntário que é realizado por pessoas que trabalham na ThoughtWorks, que introduz a pessoas a partir de 14 anos conceitos de lógica de programação e outros conteúdos relacionados a tecnologia da informação. Porém, o Educodar não é um curso.
        </div>
      </dd>
      <hr
        className="MuiDivider-root"
      />
    </div>
    <div
      className="MuiBox-root MuiBox-root-46"
      itemProp="mainEntity"
      itemScope={true}
      itemType="https://schema.org/Question"
    >
      <dt
        className="MuiBox-root MuiBox-root-47"
      >
        Tenho que pagar para estudar no Educodar?
      </dt>
      <dd
        className="MuiBox-root MuiBox-root-48"
        itemProp="acceptedAnswer"
        itemScope={true}
        itemType="https://schema.org/Answer"
      >
        <div
          className="MuiBox-root MuiBox-root-49"
          itemProp="text"
        >
          Não, ele é 100% gratuito.
        </div>
      </dd>
      <hr
        className="MuiDivider-root"
      />
    </div>
    <div
      className="MuiBox-root MuiBox-root-50"
      itemProp="mainEntity"
      itemScope={true}
      itemType="https://schema.org/Question"
    >
      <dt
        className="MuiBox-root MuiBox-root-51"
      >
        Como o Educodar me ajuda a ingressar no mercado de trabalho?
      </dt>
      <dd
        className="MuiBox-root MuiBox-root-52"
        itemProp="acceptedAnswer"
        itemScope={true}
        itemType="https://schema.org/Answer"
      >
        <div
          className="MuiBox-root MuiBox-root-53"
          itemProp="text"
        >
          <p>
            Se você for menor de idade e, portanto, ainda estiver na época da escola, nossa preocupação não será em arrumar um emprego pra você. Nós vamos mostrar a você que a área de tecnologia é uma possível oportunidade para o seu futuro.
          </p>
          <p>
            Se você é maior de idade e está procurando uma oportunidade de trabalho, não podemos garantir que você saia daqui com um emprego. Porém, vamos te apoiar e te dar direcionamento em relação a currículo, processos seletivos e possíveis cursos.
          </p>
        </div>
      </dd>
      <hr
        className="MuiDivider-root"
      />
    </div>
    <div
      className="MuiBox-root MuiBox-root-54"
      itemProp="mainEntity"
      itemScope={true}
      itemType="https://schema.org/Question"
    >
      <dt
        className="MuiBox-root MuiBox-root-55"
      >
        Tenho que fazer provas e tirar nota boa?
      </dt>
      <dd
        className="MuiBox-root MuiBox-root-56"
        itemProp="acceptedAnswer"
        itemScope={true}
        itemType="https://schema.org/Answer"
      >
        <div
          className="MuiBox-root MuiBox-root-57"
          itemProp="text"
        >
          Não. Pra gente, o mais importante é que todas as pessoas aprendam. Seu desempenho será acompanhado durante as aulas, e sempre vai ter alguém para te dar apoio se você tiver alguma dúvida.
        </div>
      </dd>
      <hr
        className="MuiDivider-root"
      />
    </div>
    <div
      className="MuiBox-root MuiBox-root-58"
      itemProp="mainEntity"
      itemScope={true}
      itemType="https://schema.org/Question"
    >
      <dt
        className="MuiBox-root MuiBox-root-59"
      >
        Como funciona o Educodar?
      </dt>
      <dd
        className="MuiBox-root MuiBox-root-60"
        itemProp="acceptedAnswer"
        itemScope={true}
        itemType="https://schema.org/Answer"
      >
        <div
          className="MuiBox-root MuiBox-root-61"
          itemProp="text"
        >
          Nós dividimos o Educodar em duas fases. Na primeira, fazemos um 
          <em
            lang="en"
          >
            workshop
          </em>
           super divertido, que vai ajudar você a aprender conceitos básicos de programação (como lógica de programação, por exemplo). Na segunda fase, você aprenderá a programar em aulas que acontecem semanalmente.
        </div>
      </dd>
      <hr
        className="MuiDivider-root"
      />
    </div>
    <div
      className="MuiBox-root MuiBox-root-62"
      itemProp="mainEntity"
      itemScope={true}
      itemType="https://schema.org/Question"
    >
      <dt
        className="MuiBox-root MuiBox-root-63"
      >
        O que aprenderei no Educodar?
      </dt>
      <dd
        className="MuiBox-root MuiBox-root-64"
        itemProp="acceptedAnswer"
        itemScope={true}
        itemType="https://schema.org/Answer"
      >
        <div
          className="MuiBox-root MuiBox-root-65"
          itemProp="text"
        >
          No início, você aprenderá 3 linguagens que são utilizadas para construir sites e até alguns aplicativos. Elas são o HTML, CSS e JavaScript.
           
        </div>
      </dd>
      <hr
        className="MuiDivider-root"
      />
    </div>
    <div
      className="MuiBox-root MuiBox-root-66"
      itemProp="mainEntity"
      itemScope={true}
      itemType="https://schema.org/Question"
    >
      <dt
        className="MuiBox-root MuiBox-root-67"
      >
        Não sei nada de informática ou programação. Posso me candidatar?
      </dt>
      <dd
        className="MuiBox-root MuiBox-root-68"
        itemProp="acceptedAnswer"
        itemScope={true}
        itemType="https://schema.org/Answer"
      >
        <div
          className="MuiBox-root MuiBox-root-69"
          itemProp="text"
        >
          Sim. Todas as pessoas são bem-vindas.
        </div>
      </dd>
      <hr
        className="MuiDivider-root"
      />
    </div>
    <div
      className="MuiBox-root MuiBox-root-70"
      itemProp="mainEntity"
      itemScope={true}
      itemType="https://schema.org/Question"
    >
      <dt
        className="MuiBox-root MuiBox-root-71"
      >
        Eu não tenho um computador. Posso participar?
      </dt>
      <dd
        className="MuiBox-root MuiBox-root-72"
        itemProp="acceptedAnswer"
        itemScope={true}
        itemType="https://schema.org/Answer"
      >
        <div
          className="MuiBox-root MuiBox-root-73"
          itemProp="text"
        >
          Sim. Enquanto você estiver em nosso local de aprendizado, você poderá usar um computador que ficará reservado para você. Porém, nós não podemos emprestar o computador para você levar para casa.
        </div>
      </dd>
      <hr
        className="MuiDivider-root"
      />
    </div>
    <div
      className="MuiBox-root MuiBox-root-74"
      itemProp="mainEntity"
      itemScope={true}
      itemType="https://schema.org/Question"
    >
      <dt
        className="MuiBox-root MuiBox-root-75"
      >
        O Educodar é um processo de recrutamento?
      </dt>
      <dd
        className="MuiBox-root MuiBox-root-76"
        itemProp="acceptedAnswer"
        itemScope={true}
        itemType="https://schema.org/Answer"
      >
        <div
          className="MuiBox-root MuiBox-root-77"
          itemProp="text"
        >
          Não. O programa Educodar não tem vínculo com nenhum processo de recrutamento. Porém, ele pode te ajudar indiretamente a se sair melhor em algum processo.
        </div>
      </dd>
      <hr
        className="MuiDivider-root"
      />
    </div>
  </dl>
  <a
    aria-disabled={false}
    className="MuiTypography-root MuiLink-root MuiLink-underlineNone MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiTypography-colorPrimary"
    href="/quero-aprender"
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex={0}
    target="_self"
  >
    <span
      className="MuiButton-label"
    >
      Quero aprender a programar
    </span>
  </a>
</article>
`;
